File: BOJ/NO.11945/string.cpp
/*
	문제 :  www.acmicpc.net/problem/11945
	문자열 배열 생성 후 입력 받아 반대부터 출력 
*/
#include<iostream>
#include<string>
using namespace std;
string arr[12];
int main()
{
	int n,m;
	cin>>n>>m;
	for(int i=0;i<n;i++)
		cin>>arr[i];
	for(int i=0;i<n;i++)
	{
		for(int j=m-1;j>=0;j--)
			cout<<arr[i][j];
		cout<<endl;
	}
	return 0;
}

File: BOJ/NO.7366/string.kt
/*
  문제 : https://www.acmicpc.net/problem/7366
  풀이 : 입력된 문자열에서 sheep (Sheep X!)의 갯수를 출력하는 간단한 문제이다.
*/
import java.util.*
import kotlin.system.exitProcess

fun main(args: Array<String>) = with(Scanner(System.`in`))
{
    var n = nextInt()
    for(z in 1..n)
    {
        var a = nextInt()
        var cnt = 0
        while(a-->0)
        {
            var s = next()
            if(s=="sheep") cnt++
        }
        println("Case $z: This list contains $cnt sheep.\n")
    }
}
